manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml

```
@@ -0,0 +1,9 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
```

```suggestion
PackageVersion: 1.3.1.0
```
This must match Control Panel.